openedx/adg/lms/applications/tests/test_admin/test_user_application_adg_admin.py

```diff
@@ -178,15 +224,21 @@ def test_changeform_view_post_with_status(
     """
     Test the overridden changeform_view.
 
-    Test that if a POST request is made with internal note and status, the application should be updated and saved.
+    Test that if a POST request is made with internal note and status, the application should be updated, saved  and
```

Comment:
It seems there is a double space between "saved " and "and".

Reply:
Done